common/static/common/js/spec/main_requirejs.js

```diff
@@ -153,6 +155,10 @@
         }
     });
 
+    // None of the common source files require jquery-migrate
+    // (it is loaded by LMS via base_vendor_js, and CMS by common_libraries).
+    require(['jquery-migrate']);
```

Comment:
But it still needs to be called before the tests run. I don't see how the refactoring would help, and it would just obfuscate what is in spec_dependencies. Given that this file contains the RequireJS configuration for running the unit tests, I think this is a natural place to require libraries.